### ISSUE TITLE
Follow Giscus' MathJax-based math rendering?

### ISSUE THREAD

**Issue by AustenLamacraft** [tripos-education/maths-tripos-questions#244]
[Giscus now supports math rendering](https://github.com/giscus/giscus/discussions/62#discussioncomment-2796378) using MathJax, like GH itself. Should we switch out the (KaTeX based) rendering of comments and track Giscus / GH closely?

**Comment by thenewtonium** [comment]
Does MathJax have an equivalent to `copy-tex`?

**Comment by AustenLamacraft** [comment]
Yeah they have this [contextual menu](https://docs.mathjax.org/en/latest/options/menu.html) which includes copy to clipboard as an option. Slightly clunkier, but also more explicit

**Comment by thenewtonium** [comment]
Epic
Then I don't see anything against it, as long as it doesn't break anything 🤷
